core/components/import/LvImportFileSelect.vue

```
@@ -0,0 +1,39 @@
+<template>
+  <LvInputFileDrop @select="selectFile" :label="$l('import_drag')">
+    <LvInputFileSelect @select="selectFile" accept=".dxf,.pdf" class="py-2" :label="$L('import_select')"/>
+    <div>{{ filename }}&nbsp;</div>
+  </LvInputFileDrop>
+</template>
+
+<script setup>
+import LvInputFileDrop from './LvInputFileDrop.vue';
+import LvInputFileSelect from './LvInputFileSelect.vue';
+import useShapeSelect from '../../composables/useShapeSelect';
+import { ref } from 'vue';
+
+const { setShapesFromFile } = useShapeSelect();
+
+const filename = ref('');
+
+async function selectFile(file) {
+  const { name } = file;
+  const type = name.split('.').pop();
+  filename.value = name;
+  const content = await getFileContent(file);
```

do we handle some possible errors? not sure if it's a real case though, but if some corrupted file or wrong format still, will it just fail silently or throw some error?